org/cloudfoundry.yml

```diff
@@ -111,7 +111,6 @@ orgs:
     - bsoroushian
     - bwinter
     - C-Otto
-    - cagiti
```

Comment:
This should be resolved once we get a successful run. Related change: https://github.com/cloudfoundry/community/pull/211